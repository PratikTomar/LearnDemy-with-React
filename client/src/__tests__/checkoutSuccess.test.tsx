
import React from 'react';
import { render, screen } from '@testing-library/react';
import CheckoutSuccess from '../components/checkout/checkoutSuccess.component';

test('renders transaction success message', () => {
 render(<CheckoutSuccess />);
  const successMessage = screen.getByText('Transaction Successful');
  expect(successMessage).toBeInTheDocument();
  expect(successMessage).toMatchSnapshot();
});

