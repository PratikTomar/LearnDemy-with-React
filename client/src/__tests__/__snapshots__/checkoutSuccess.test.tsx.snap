// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders transaction success message 1`] = `
<div
  class="checkout-success-container"
>
  Transaction Successful
</div>
`;
